postreise/plot/plot_shadowprice_map.py

```diff
@@ -297,34 +284,21 @@ def _construct_bus_legend(lmp_split_points):
 
 
 def _get_bus_legend_bars_and_labels(lmp_split_points, x_range):
-    """Gets the bar lengths and labels for the bus legend
-
-    :param lmp_split_points: the lmp vs we have chosen to split the bus data
-    :type lmp_split_points: list(float)
-    :param x_range: the x-range for the vbar_stack
-    :type x_range: list(string)
-    :return: bar lengths and labels for the bus legend
-    :rtype: dict, float, dict
+    """Get the bar lengths and labels for the bus legend
+
+    :param list/tuple/set/numpy.array lmp_split_points: lmp values chosen to split
+        the bus data
+    :param list x_range: the x-range for the vertical bar
```

Comment:
It seems that this parameter is never actually used in this function, and the relevant info could be added to the `bars` dict within `_construct_bus_legend`, after the return from `_get_bus_legend_bars_and_labels`.

Reply:
Yes I saw that. Same thing with `bar_length_sum`, it is equal to `max(bars.values)` and hence does not need to be returned.

Reply:
Done